### ISSUE TITLE
Move docs into wiki

### ISSUE THREAD

**Issue by freddinator** [rappasoft/laravel-5-boilerplate#23]
I've noticed the readme file has become very messy with the recent additions to the boilerplate. I have a bit of free time at the moment, so (If it's okay with you @rappasoft) I'll migrate some stuff into the wiki.

**Comment by rappasoft** [comment]
I would be very grateful.

**Comment by rappasoft** [comment]
Use the 5.1 docs, they are different and will be merged in as soon as that goes live.

**Comment by rappasoft** [comment]
I do have http://laravel-boilerplate.com redirecting to here, eventually we will have a real website, once I get time.

**Comment by Teranode** [comment]
Just a genric out of place question i'm asking most people, but do you know if there are any packages thath as social groups (groups people can join). Doesnt have to be advanced just something i can start off with.

**Comment by rappasoft** [comment]
Use this, add a flag to the roles table called social_group, pull them into the frontend as a dropdown on the register screen. Boom, people adding themselves to specific roles.

**Comment by Teranode** [comment]
well it's kind of like your thing except the user is able to make their own social group and set their own user roles (like rankings) and permissions. Within their own group other users can join, or request to join and then there will be like a little feed. The owner of the group is allowed to make a public group or private group.

**Comment by rappasoft** [comment]
Haha that's a lot more complicated than you lead to believe. It's doable with this, is it the best solution? I'm not sure.

**Comment by Teranode** [comment]
Well I have a schematic in my head on how it would function. It generally is the same thing as your roles/permission but more on the user end of things than the admin. I know it's complicated >_>.

If it were to be made, it would be like adding a second access facade but for users.

**Comment by Teranode** [comment]
in the AccessTable Seeder, how would you ad multiple permissions to the Administrator role?

**Comment by Teranode** [comment]
like a mass assignable method

**Comment by rappasoft** [comment]
It takes an array just pass multiple permission id's.

**Comment by Teranode** [comment]
so i have to reicreae each permission model instance?

**Comment by rappasoft** [comment]
```
$admin->permissions()->sync(
			[
				$viewAdminLink->id,
                                $perm2->id,
                                $perm3->id
			]
		);
```

**Comment by rappasoft** [comment]
I'll continue the wiki when I have a chance. I just released 5.1 because i'm tired of waiting. I'll just change the min-stability when its launched.
